editor/src/messages/portfolio/document/node_graph/node_graph_message_handler.rs

```diff
@@ -42,22 +42,26 @@ pub struct NodeGraphMessageHandler {
 	begin_dragging: bool,
 	/// Stored in node graph coordinates
 	box_selection_start: Option<DVec2>,
+	/// Restore the selection before box selection if it is aborted
+	selection_before_pointer_down: Vec<NodeId>,
 	/// If the grip icon is held during a drag, then shift without pushing other nodes
 	shift_without_push: bool,
 	disconnecting: Option<InputConnector>,
 	initial_disconnecting: bool,
 	/// Node to select on pointer up if multiple nodes are selected and they were not dragged.
 	select_if_not_dragged: Option<NodeId>,
-	/// The start of the dragged line that cannot be moved, stored in node graph coordinates
+	/// The start of the dragged line (cannot be moved), stored in node graph coordinates
 	pub wire_in_progress_from_connector: Option<DVec2>,
-	/// The end point of the dragged line that can be moved, stored in node graph coordinates
+	/// The end point of the dragged line (cannot be moved), stored in node graph coordinates
```

Comment:
Should this say "cannot" or "can"? It previosly said "can".

Reply:
it should say cannot because you cannot move the wire from the output port